app/modules/enduser_setup.c

```diff
@@ -626,6 +626,52 @@ static void do_station_cfg (task_param_t param, uint8_t prio)
 }
 
 
+/**
+ *
+ *
+ *
+ *
+ */
+static int enduser_setup_write_file_with_extra_configuration_data(
+char * configuration_string, int configuration_length
+)
+{
+  int p_file = 0;
+  lua_State *L = NULL;
```

Comment:
Stylistically I'd prefer to see these declared right before use rather than old-C style top-of-function.